### PR TITLE
dcerpc: add app-layer metadata in alerts

### PR DESCRIPTION
**Pull request by catenacyber**
Link to ticket: https://redmine.openinfosecfoundation.org/issues/
https://redmine.openinfosecfoundation.org/issues/6090

Describe changes:
- https://redmine.openinfosecfoundation.org/issues/6090

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1929

### CODE REVIEW

File: src/output-json-alert.c [hunk 1]
```diff
@@ -381,6 +381,26 @@ static void AlertAddAppLayer(const Packet *p, JsonBuilder *jb,
                 jb_restore_mark(jb, &mark);
             }
             break;
+        case ALPROTO_DCERPC:
+            jb_get_mark(jb, &mark);
```
Comment (victorjulien) [inline src/output-json-alert.c:385]: nit: could call this a bit later

**Review by victorjulien — APPROVED** [review]

**Comment by suricata-qa** [comment]
ERROR:
 

ERROR: QA failed on SURI_TLPW2_autofp_suri_time.
  

Pipeline 21156

**Comment by suricata-qa** [comment]
Information:
 

ERROR: QA failed on SURI_TLPW2_autofp_suri_time.

ERROR: QA failed on SURI_TLPW1_suri_time.
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPW2_autofp_stats_chk | | |
| .uptime | 137 | 143 | 104.38% |
| | SURI_TLPR1_stats_chk | | |
| .uptime | 642 | 663 | 103.27% | 

Pipeline 21157

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 21165

**Comment by victorjulien** [comment]
Merged in #11353, thanks!

**Pull request closed without merging**